samples/apps/research-assistant-react/README.md

```diff
@@ -78,8 +74,8 @@ The agents responds by _writing code_ to create a python program to generate the
 
 ## FAQ
 
```

Comment:
Add: How do I change the model?

Reply:
Good point. Added a note on this to the FAQ

- How do I specify the agent configuration (e.g. temperature, model, agent system message etc). You can do either from the UI interface or by modifying the default agent configuration in `utils.py`

Also added support for this in the UI. 
<img width="1463" alt="image" src="https://github.com/pcdeadeasy/autogen-oss/assets/1547007/0f2112de-9a75-4ce1-a257-12b69aba1f74">